src/js/vic-v2/config/form.js

```diff
@@ -235,7 +240,7 @@ const formConfig = {
                 'bmp'
               ],
               maxSize: TWENTY_FIVE_MB,
-              hideLabelText: true,
+              alternativeLabel: 'Upload your discharge document',
```

Comment:
Is this used?

Reply:
This is intended to update the button text, I've updated the File field and used buttonText instead in an effort to make the intended use clearer.